### DISCUSSION TITLE
How to delete cookie？

### DISCUSSION THREAD

**Discussion by 9death** [cefsharp/CefSharp#4423] [General]
My Cef version is 65.0,

            Cef.GetGlobalCookieManager().DeleteCookies("", "");
            Cef.GetBlockingCookieManager().DeleteCookies("", "");

The above two sentences are called, but it doesn't work

**Comment by amaitland** [comment]
In future please use [Discussions](https://github.com/cefsharp/CefSharp/discussions) to ask support related  questions.

> My Cef version is 65.0,

I'm only providing limited support for old versions on a **paid** basis. If you require further assistance I'd be happy to discuss your problem in more detail.

> The above two sentences are called, but it doesn't work

It's impossible to know what from lines what the problem is. It is important to remember that cookes are deleted async, they're not deleted instantly. Web pages needed to be reloaded also.

API Docs are available at https://cefsharp.github.io/api/ I'd suggest picking the closes matching version and reviewing the documentation. There are two methods of deleting cookies via the `ICookieManager`, you can potentially try the other method.

There's a xunit test available for reference https://github.com/cefsharp/CefSharp/blob/master/CefSharp.Test/CookieManager/CookieManagerTests.cs#L184
(This will be relevant for the current supported version).
